src/worker/background.ts

```diff
@@ -0,0 +1,8 @@
+chrome.webRequest.onCompleted.addListener(
+    (details) => {
+      chrome.storage.sync.remove("os-access-token");
+    },
+    { urls: ["https://ibcwmlhcimymasokhgvn.supabase.co/auth/v1/logout"] }
+  );
+  
+export {};
```

Comment:
Does it not work without this line?

Reply:
The export line? The file is declared as a module in the manifest. Hence, it must contain at least one import or export to be built in production.